src/dashboard/alerts/alert.py

```diff
@@ -64,20 +40,24 @@ def __init__(self, pkt: Packet, alert_description="", alert_type=ALERT_TYPE.UNKN
         self.type = str(alert_type)
         self.severity = int(alert_severity)
 
-        if is_destination:
-            if "IP" in pkt:
-                self.device_ip = pkt["IP"].dst
-            if "Ethernet" in pkt:
-                self.device_mac = pkt["Ethernet"].dst
-        else:
-            if "IP" in pkt:
-                self.device_ip = pkt["IP"].src
-            else:
-                self.device_ip = "[Layer 2]"
-            if "Ethernet" in pkt:
-                self.device_mac = pkt["Ethernet"].src
+        if pkt:
+            if is_destination:
+                if "IP" in pkt:
+                    self.device_ip = pkt["IP"].dst
+                if "Ethernet" in pkt:
+                    self.device_mac = pkt["Ethernet"].dst
             else:
-                self.device_mac = "[Unknown]"
+                if "IP" in pkt:
+                    self.device_ip = pkt["IP"].src
+                else:
+                    self.device_ip = "[Layer 2]"
+                if "Ethernet" in pkt:
+                    self.device_mac = pkt["Ethernet"].src
+                else:
+                    self.device_mac = "[Unknown]"
+            self.payload_info = raw(pkt)
+        else:
+            self.payload_info = "None"
 
         # Use some magic config trickery to get the name for this device, otherwise unknown
         if "magic":
```

Comment:
What is this line supposed to be doing?

Reply:
We wanted to have some form of a device name/classification (Camera, Light Strip 2, etc.) added to alerts for ease of use, and this is a placeholder for that. If we no longer plan on including this I can remove it, but I assumed we were still going down this route once we found some way to pull hostnames or resolve MAC addresses some how.

Reply:
Can this just be replaced with a TODO: then so it flags in our IDEs instead of the conditional here?

Reply:
I can do that.